pennylane_lightning_gpu/src/util/MPIManager.hpp

```diff
@@ -424,6 +431,16 @@ class MPIManager final {
                                      this->getComm()));
     }
 
+    template <typename T>
+    void Reduce(DataBuffer<T> &sendBuf, DataBuffer<T> &recvBuf, size_t length,
```

Comment:
Can we add a docstring for this function?

Reply:
Done!